### PR TITLE
fix(formatter): Rename and export the main class

### LINKED ISSUE #632: Make pretty-printer available to third-party tools

### PR DESCRIPTION
**Pull request by byakuren-hijiri**
Closes #632

- [ ] ~~I have updated CHANGELOG.md~~ (since there were no an entry introducing the formatter)
- ~~`[ ] I have documented my contribution in Tact Docs: https://github.com/tact-lang/tact-docs/pull/PR-NUMBER~~
- ~~[ ] I have added tests to demonstrate the contribution is correctly implemented: this usually includes both positive and negative tests, showing the happy path(s) and featuring intentionally broken cases~~
- [ ] I have run all the tests locally and no test failure was reported
- [ ] ~~I have run the linter, formatter and spellchecker~~
- [x] I did not do unrelated and/or undiscussed refactorings

### CODE REVIEW

File: src/formatter.ts [hunk 1]
```diff
@@ -44,9 +44,9 @@ import {
 } from "./grammar/ast";
 
 /**
- * PrettyPrinter class provides methods to format and indent Tact code.
+ * Formatter provides methods to format and indent Tact code.
  */
-class PrettyPrinter {
+export class Formatter {
```
Comment (anton-trunov) [inline src/formatter.ts]: Hmmm, technically this is a printer, because formatter is a parser + printer. Perhaps we should rename the file?

Comment (byakuren-hijiri) [reply src/formatter.ts]: `prettyPrinter.ts`? `pp.ts`?

Comment (anton-trunov) [reply src/formatter.ts]: let's go with `prettyPrinter.ts`

**Comment by byakuren-hijiri** [comment]
There are some problems with the `prettyPrint` function. I'll double-check everything while working on it in Misti and update the PR afterward.

**Review by anton-trunov — APPROVED** [review]

**Pull request merged**